shotover/src/codec/cassandra.rs

```diff
@@ -139,8 +139,10 @@ impl CodecBuilder for CassandraCodecBuilder {
         )
     }
 
-    fn websocket_subprotocol(&self) -> &'static str {
-        "cql"
+    fn protocol(&self) -> ProtocolType {
```

Comment:
Could we pull the `ProtocolType` changes out of this?

Reply:
Are you thinking to make the ProtocolType a prereq?
It is a requirement of the `TransformContextConfig`.

Reply:
Yep, thanks. I might as a pre req

Reply:
sure sounds reasonable,